salt/states/pkgrepo.py

```diff
@@ -307,7 +307,15 @@ def managed(name, ppa=None, copr=None, **kwargs):
        :mod:`pkg.latest <salt.states.pkg.latest>` to trigger the
        running of ``apt-get update`` prior to attempting to install these
        packages. Setting a require in the pkg state will not work for this.
+
+    aptkey: Use the binary apt-key
+
+    signedby:
```

Comment:
How does this interact with `name`?
Can you put it in the name instead?
Does this override the one in the name if present?